pkg/namespace/namespace.go

```diff
@@ -127,6 +131,18 @@ func GetAllNamespaces(endPoint string) (namespaces AllNamespaces, err error) {
 	return
 }
 
+// GetNamespaces takes a Kubernetes clientset and returns all namespaces with type *v1beta1.IngressList and an error.s
+func GetAllNamespacesFromCluster(clientSet *kubernetes.Clientset) ([]v1.Namespace, error) {
+
+	namespaces, err := clientSet.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		log.Fatalln("Can'list namespaces from cluster", err.Error())
+		return nil, err
```

Comment:
Can you either use the log.Fatalln oruse the err return type, not both.

Reply:
Updated the code